src/Libraries/Microsoft.Extensions.AI.OpenAI/OpenAIChatClient.cs

```diff
@@ -623,11 +623,10 @@ private static ChatCompletionOptions ToOpenAIOptions(ChatOptions? options)
     /// <summary>Converts an Extensions function to an OpenAI chat tool.</summary>
     private static ChatTool ToOpenAIChatTool(AIFunction aiFunction)
     {
-        // Default strict to true, but allow to be overridden by an additional Strict property.
-        bool strict =
-            !aiFunction.AdditionalProperties.TryGetValue("Strict", out object? strictObj) ||
-            strictObj is not bool strictValue ||
-            strictValue;
+        bool? strict =
+            aiFunction.AdditionalProperties.TryGetValue("Strict", out object? strictObj) &&
```

Comment:
Do you think we should have a more specific name for this? It's not obvious what it means for the function itself to be "strict", whereas if we called it `StrictJsonSchema` there would be no ambiguity.

Reply:
Possibly, but wouldn't this break folks relying on the current name to set it?

Reply:
We can make breaking changes before GA :) Not saying it's not something to think carefully about, but it is allowed.

Personally I'd prefer taking a breaking change on this relatively niche API in preview if that gets us the API shape we want to support forever.

Reply:
BTW we are doing breaking changes for loads of stuff related to `AdditionalProperties` in other places because we're greatly reducing its usage for configuring requests and getting custom properties from responses. This would be a minor change by comparison.

Reply:
Fair. I'll rename to `stringJsonSchema` then.